app/models/template.rb

```diff
@@ -1,7 +1,8 @@
 class Template < ActiveRecord::Base
   validates_lengths_from_database
 
-  attr_accessible :name, :template, :snippet, :audit_comment, :location_ids, :organization_ids, :locked, :default
```

Comment:
As it's in the API docs then it'd need to be added back to permit mass-assignment.